evalml/automl/utils.py

```diff
@@ -261,3 +262,62 @@ def get_pipelines_from_component_graphs(
             ),
         )
     return created_pipelines
+
+
+def get_threshold_tuning_info(automl_config, pipeline):
```

Comment:
I stuck these in automl utils instead of putting them in the engine_base file/making a new engine utils file, because part of their use is in determining after automl search whether pipelines were trained on the full data. So since they can be used in the wider context of automl, I thought it made sense to put them there. If anyone has any strong feelings to the contrary, definitely let me know!